### ISSUE TITLE
Fastclick targets for constructionsite windows are off

### ISSUE THREAD

**Issue by Noordfrees** [widelands/widelands#5097]
Please fill out the relevant sections below and delete the rest.

**Describe the bug**
The fastclick targets (the place where the mouse cursor jumps to automatically when a window opens) are incorrect for constructionsite windows.

**To Reproduce**
1. Click on a half-completed own constructionsite, especially for a building that costs several ware types and only has one ware type left to be shown in its window (in the image below, an amazons tower that lacks only ironwood).
2. The mouse should be placed in the centre of the workarea button, but it is actually located a bit below.

**Expected behavior**
The mouse should be placed in the centre of the workarea button

**Screenshots**
After clicking on (1), the mouse jumps to (2) but it should jump to (3):
![grafik](https://user-images.githubusercontent.com/48293446/138707171-997d6107-7235-4cad-9843-cbb1cd66ac6f.png)

**Version:**
 - OS: Ubuntu Mate 21.10
 - Widelands Version: 1.1~git25486[0c381ba@master] (Release)

**Additional context**
Perhaps invisible input queue displays are taken into account when calculating the fastclick panel's absolute position?
